### PR TITLE
[@babel/standalone] Remove additional function context

### PR DESCRIPTION
**Pull request by stevefan1999**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            |  https://github.com/babel/babel/issues/7093
| Patch: Bug Fix?          | Y
| Major: Breaking Change?  | N
| Minor: New Feature?      | Y
| Tests Added + Pass?      | N/A
| Documentation PR         | N/A
| Any Dependency Changes?  | N/A
| License                  | MIT

### CODE REVIEW

**Comment by nicolo-ribaudo** [comment]
Looking at the comment about the `transformScriptTags` function, it shuld be called with no arguments the first time so that every `<script>` tag is transformed:
https://github.com/stevefan1999/babel/blob/9c4cb9a320c28137c31b413adc285dfba81d5d66/packages/babel-standalone/src/index.js#L255-L261

**Comment by stevefan1999** [comment]
@nicolo-ribaudo Yes, https://github.com/babel/babel-standalone/pull/97#pullrequestreview-85367986 pointed on the same finger. I was on my way to make a quickfix for that, somehow I felt asleep...

**Comment by Daniel15** [comment]
This looks good to me. Did you test all the examples and ensure they still work? https://github.com/babel/babel/tree/master/packages/babel-standalone/examples

**Comment by nicolo-ribaudo** [comment]
@stevefan1999 Can you run `make lint` to fix the linting errors?

**Comment by stevefan1999** [comment]
`scriptTag-disabled.htm` should be able to describe the situation I'm into. Try doing that in an older `babel-standalone` package.

**Comment by stevefan1999** [comment]
@existentialism Using an older version of `babel-standalone` in `scriptTag-disabled.htm` will overwrite the node even with `Babel.disableScriptTags()` disabled beforehand. Under this context, this is obviously not what we wanted!

tl;dr `Babel.disableScriptTags()` actually had never been worked before.

**Comment by Daniel15** [comment]
Looks good to me! A unit test would be good too, but this is totally fine.

> tl;dr Babel.disableScriptTags() actually had never been worked before. 

It worked back when it was originally added 😛  It must have broken at some point later on, when an argument was added to `transformScriptTags`
